### ISSUE TITLE
Oppia Blurbs

### ISSUE THREAD

**Issue by stepmalt** [oppia/design-team#126]
**Platform**
Oppia Web 

**Describe the request**
Auditing the Oppia blurbs, editing, and rewriting for clarity, conciseness, and lack of grammatical errors. Keeping in mind audience, tone, and intended purpose.

**Context for the request**

This will be for current Oppia blurbs across various platforms, which need tightening.

**Link to PRD**

MAIN:
https://docs.google.com/spreadsheets/d/12m0HEUWknpZq32tD7_OISMm8ISuMDKnYNuMxg7LW-sc/edit#gid=0

SUPPLEMENTAL INFO:

https://docs.google.com/document/d/1gtTqbsRD72xdzvYFmRoLZh6taYtOy50MwZUZC3FLPNs/edit

**Information about deadlines/priorities**
To be completed by the end of February.

**DESIGN APPROVAL CHECKLIST -- DO NOT EDIT.**

- [ ] UX Design approved (Android mocks should include dark mode)
- [ ] UX Writing approved
- [ ] Design Systems Team approved
- [ ] Product approved

_For a design project to be completed, it needs approval from UXD, UXW, DS and PM. Please tag the following reviewers when your project is ready (at least one reviewer for each of the 4 bullet points below):_
- UXD: @rflore / @S4v8n (for Web projects)
- UXW: @stepmalt
- Design systems: @qluuu / @celestinetales
- PM: @seanlip

**Comment by stepmalt** [comment]
For this first milestone, @tydesignit, can you work on the section titled "potential partners"? 

Partnership collateral material
Digital Impact Alliance vendors (Sean got an invitation to create an account)

I am not sure about these tasks, so you'll have to direct questions about them to Sean so he can explain more! And if you could cc me on any of those emails that would be great! I can help where I can, but some projects, like this, I am unsure of the specific needs.

**Comment by tydesignit** [comment]
> For this first milestone, [@tydesignit](https://github.com/tydesignit), can you work on the section titled "potential partners"?
> 
> Partnership collateral material Digital Impact Alliance vendors (Sean got an invitation to create an account)
> 
> I am not sure about these tasks, so you'll have to direct questions about them to Sean so he can explain more! And if you could cc me on any of those emails that would be great! I can help where I can, but some projects, like this, I am unsure of the specific needs.

Yep will do!

**Comment by stepmalt** [comment]
https://docs.google.com/document/d/1gtTqbsRD72xdzvYFmRoLZh6taYtOy50MwZUZC3FLPNs/edit


This may be helpful for this project, if you wanted to take a look at this or use it for future milestones! @tydesignit

**Comment by stepmalt** [comment]
https://docs.google.com/document/d/1eWBd2fXNPPKvUjcWJsS0ApgCCyiIX_bmi3D31xP1zYI/edit?usp=sharing



Hi @seanlip - I tagged you in the Google Doc, but unsure if it went through. It may have. If so, sorry for the multiple tags it may have sent you as I was trying to figure it out. If there's any way you can respond to some of my comments in the document, let me know! Absolutely no rush, this is not pressing. Thanks for your time!

**Comment by seanlip** [comment]
Thanks @stepmalt for the note! The tag did not go through -- you need to use my full email for that, not just the part after the "@" bit. It should probably autocomplete if you are doing it right.

In any case, I've got the note now and will check the doc. Thanks!

**Comment by tydesignit** [comment]
Currently working on:

Google search snippets

**Comment by stepmalt** [comment]
Update:

I just finished the Play Store description and the Oppia Foundation page.

**Comment by tydesignit** [comment]
@seanlip I have a question about (Google search snippets (possibly SEO more generally)) section.

Is this dealing with the results presented when someone searches Oppia on Google? (See examples)

![image](https://github.com/oppia/design-team/assets/47770536/555b8562-8dc0-43ce-813f-7012ccca34f6)

![image](https://github.com/oppia/design-team/assets/47770536/9e882b33-f241-4ec9-92e9-d0561d76268f)

Or is this more broadly for other searches within Google? It is my understanding that these snippets are provided for when someone gets an answer from an article or web page (eg. what are the best cookie recipes?). Then results are displayed at the top of the search. I am unsure if that applies to Oppia but correct me if I am wrong.

![image](https://github.com/oppia/design-team/assets/47770536/1e9caf17-6417-43fc-8d2e-6ac19241c726)

**Comment by seanlip** [comment]
Hi @tydesignit, it's just the first one. Thanks!

**Comment by stepmalt** [comment]
Hi @seanlip ! On the Google Doc, LinkedIn Bio is listed under two categories, and I just noticed it has been rewritten twice. Is this a mistake that it was listen in two different categories? If so, we will condense and only go with one version. Thank you for your time!

**Comment by stepmalt** [comment]
Hi Kwami! Here is the main document we're working out of:

https://docs.google.com/document/d/1eWBd2fXNPPKvUjcWJsS0ApgCCyiIX_bmi3D31xP1zYI/edit?usp=sharing

If you could work on the newsletter welcome series, that would be a big help! I am not entirely familiar with this, so I would reach out to Sean to get more information about the project before you start drafting. And if you need anything regarding the copy or brainstorming, let me know! But Sean will know more about what exactly he's hoping for concerning that project.

**Comment by seanlip** [comment]
Hi @stepmalt -- thanks, I'm not sure why the Linkedin bio is in the Donate section. We can remove that one and keep the other.

For the newsletter welcome series, could you please check in with Jen (marketing team lead, @JNunez1204) to see whether this still needs UXW help? Thanks!

**Comment by JNunez1204** [comment]
I don't understand where the emails come into this- what is this for, the website? What's the email reference about exactly?

**Comment by seanlip** [comment]
> I don't understand where the emails come into this- what is this for, the website? What's the email reference about exactly?

@JNunez1204 Did our discussion just now help clarify this?

**Comment by stepmalt** [comment]
@JNunez1204 Let me know if based on your convo with Sean if it still needs UXW or not, so I can assign a different project to Kwami if not!

**Comment by JNunez1204** [comment]
This is the current welcome email 
![image](https://github.com/oppia/design-team/assets/56636806/c20c5461-b8b3-42de-8081-30ee1d11733c)

**Comment by stepmalt** [comment]
@JNunez1204 So it's not necessarily a welcome "series" currently, it's just this one email? 

@seanlip Would you like us to take a look at and rewrite/edit this? And this be the only email? Or do you want more emails written as part of the "series"?

**Comment by seanlip** [comment]
I think that's up to @JNunez1204 really. I just thought, for the "blurbs doc" side of things, that this is an example of a touchpoint with our users, so let's make sure it's consistent with other things. If you think the existing copy looks good and doesn't need changing, that's fine too!

**Comment by JNunez1204** [comment]
Yes @stepmalt  It's currently just one email because of our limitations with mailchimp. It's in process of potentially using another service but if we don't, then we can only send 2 emails. If you feel this email needs editing just let me know what your thoughts/changes are

**Comment by stepmalt** [comment]
Update on this project. It's nearly done, these are the projects left to tackle:


Flyers for learners/parents - done
Directories of learning tools
Newsletter welcome series- done
Newsletter signup page - done




Should be finished by next week. 

@Kwam98 If you are looking for a project and want to take one of these, let me know!

**Comment by seanlip** [comment]
That's awesome, thanks @stepmalt! Just a note: "Newsletter welcome series" and "Mailchimp welcome series" are the same thing.

**Comment by stepmalt** [comment]
@seanlip Thanks for the note! Quick question, can you give me more information on the "directories of learning tools" blurb? There's nothing there currently aside from this link https://nonprofit.yourcause.com/nonprofits/nonprofit-public-profile/5458210

Just tell me how you'd like me to go about it!

Best,

Stephanie.

**Comment by seanlip** [comment]
@stepmalt I have a feeling something got jumbled along the way, because I don't think those are supposed to be the same thing. Looking at the original doc I wrote, those are actually two separate bullet points: 

- Directories of learning tools (e.g. UNICEF etc.)
- https://nonprofit.yourcause.com/account-setup/nonprofit-stories – Stories on YourCause platform

For directories of learning tools, I think the idea here is to create a blurb that describes "Oppia in a nutshell" that we can put in various places if we're asked for it. Maybe a few versions that differ in length (2 sentences, 1 paragraph, 2 paragraphs).

For YourCause, the idea here is to put compelling stories on the [linked page](https://nonprofit.yourcause.com/nonprofits/nonprofit-public-profile/5458210) that we'd like to share with potential donors. Stories typically consist of a blurb and a picture, possibly with a link to read more detail. 

Hope this helps clarify, thanks!

**Comment by stepmalt** [comment]
Copy reviewed by @jikim322 on 4/6.

Next steps: UXW lead approval.
